### ISSUE TITLE
Debug why asyncio event loop is being closed in tests

### ISSUE THREAD

**Issue by milesgranger** [equinor/gordo#212]
https://github.com/equinor/gordo-components/pull/211 introduced a pytest fixture to ensure the event loop stays open as a work-around.

The problem showed up after refactoring test files into sub directories which matched the outline of the components in the codebase. While moving `tests/test_client.py` -> `tests/client/test_client.py` (or other sub dirs) it would error with `Event loop is closed` error. If that file is moved back tests will pass again. Furthermore, if you only run client tests, it will happily run the `tests/client/test_client.py` without errors.

**Comment by milesgranger** [comment]
No longer relevant after #587
